drivers/sensors/lsm6dso32_uorb.c

```diff
@@ -1905,6 +1905,7 @@ int lsm6dso32_register(FAR struct i2c_master_s *i2c, uint8_t addr,
 
   priv->gyro.lower.type = SENSOR_TYPE_GYROSCOPE;
   priv->gyro.lower.ops = &g_sensor_ops;
+  priv->gyro.lower.nbuffer = 10;
```

Comment:
Please "define a ODR_BUFFER_SIZE 10" in the Pre-processor Definition area instead of hard-code the value here. This way it could be easier to people discover and change this value